### PR TITLE
Hide inactive comment box tabs

### PR DESCRIPTION
**Pull request by a0m0rajab**
This PR adds a file to hide the comment in the archived issues, to have a better view. 
I checked this with PRs and issues that was not archived and did not see and breaking changes. 

close https://github.com/refined-github/refined-github/issues/7108

I could not test this because I was getting an error from nodejs: 

```bash
node: --import is not allowed in NODE_OPTIONS
ERROR: "build:webpack" exited with 9.
```
I will double check it later and see if I could get over that error. 

## Test URLs

https://github.com/liip/bund_ds/issues/132#event-10996157695

## Screenshot

| Before | After |
|--------|--------|
| image | image |

### CODE REVIEW

**Comment by a0m0rajab** [comment]
Note: the lint is unsuccessful due to files I did not touch.

File: source/features/clean-archived-issue.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.comment-form-head.tabnav {
```
Comment (fregante) [inline source/features/clean-archived-issue.css:1]: This selects all the navigation, it has no reference to archived issues

File: source/features/clean-archived-issue.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+    visibility: hidden;
```
Comment (fregante) [inline source/features/clean-archived-issue.css:2]: Use display: none

**Review by fregante — CHANGES_REQUESTED** [review]

File: source/features/clean-archived-issue.css (hunk 2 above)
Comment (a0m0rajab) [reply source/features/clean-archived-issue.css]: Done.

File: source/features/clean-archived-issue.css (hunk 1 above)
Comment (a0m0rajab) [reply source/features/clean-archived-issue.css:1]: Thanks, I will check the MDN docs: https://developer.mozilla.org/en-US/docs/Web/CSS/:has

**Review by tyrann0us — COMMENTED** [review]
I tested the CSS selector in different scenarios:
* archived repository -> editor header gone ✅
* locked issue -> editor header gone ✅
* locked issue with write permissions -> editor header present due to different HTML structure ✅

So, the feature works. However, it affects not only archived repositories but also locked issues of _non-archived repositories_. This is actually good, and I didn't have this in mind when creating #7108, but we should name the feature accordingly.

Alternatively, and I actually would prefer this, instead of adding a new CSS file, we could add the rule to https://github.com/refined-github/refined-github/blob/feat-hide-comment-area/source/features/locked-issue.css.

**Comment by tyrann0us** [comment]
> it's easier to just keep separate files and name features.

Ok. So `clean-archived-issue.css` should be renamed to `clean-locked-issue.css` or similar. CC @a0m0rajab

File: readme.md [hunk 3]
```diff
@@ -250,6 +250,7 @@ Thanks for contributing! 🦋🙌
 - [](# "jump-to-conversation-close-event") [Adds a link to jump to the latest close event of a conversation.](https://user-images.githubusercontent.com/16872793/177792713-64219754-f8df-4629-a9ec-33259307cfe7.gif)
 - [](# "close-as-unplanned") [Lets you "close issue as unplanned" in one click instead of three.](https://github-production-user-asset-6210df.s3.amazonaws.com/1402241/279745773-709cde60-c26a-4a0e-89e1-56444d25ebdf.png)
 - [](# "locked-issue") [Show a label on locked issues and PRs.](https://user-images.githubusercontent.com/1402241/283015579-0a04becc-9bff-4aef-8770-272d6804970b.png)
+- [Hides the inactive comment box tabs.](https://github.com/refined-github/refined-github/issues/7108)
```
[the remark this answers is not in the document]
Comment (fregante) [reply readme.md:253]: Actually I don’t think this needs to be documented. It's super minor

[the remark this answers is not in the document]
Comment (kidonng) [reply readme.md:253]: Ah yes, didn't notice this goes into `github-bugs`.

[the remark this answers is not in the document]
Comment (a0m0rajab) [reply readme.md:253]: > Please follow [the description style guide](https://github.com/refined-github/refined-github/blob/461eb5cf467feaf240dcd87d78873c579245473f/readme.md?plain=1#L113-L138) and add a screenshot.

Did not notice that there is a description style guide, thank you for showing me that! 

> Actually I don’t think this needs to be documented. It's super minor

I will remove this part.

**Pull request merged**